package.json

```diff
@@ -66,7 +66,7 @@
     "ethereumjs-util": "^6.1.0",
     "lodash.get": "^4.4.2",
     "nahmii-contract-abstractions": "2.0.1",
-    "nahmii-contract-abstractions-ropsten": "^3.1.0",
+    "nahmii-contract-abstractions-ropsten": "^3.1.1",
```

Comment:
Wouldn't it be safer to specify a specific version of the Ropsten contracts instead of using a wild-card?

Reply:
I think that makes sense, given that new package of contract abstractions Ropsten is not state-wise not back-ward compatible with old package. I'll update.